build/Dockerfile

```diff
@@ -1,14 +1,12 @@
 FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
-ENV OPERATOR_PATH=/go/src/github.com/openshift/configure-alertmanager-operator
-COPY . ${OPERATOR_PATH}
-WORKDIR ${OPERATOR_PATH}
+COPY . /go/src/github.com/openshift/configure-alertmanager-operator
+WORKDIR /go/src/github.com/openshift/configure-alertmanager-operator
 RUN make gobuild
 
 FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
-ENV OPERATOR_PATH=/go/src/github.com/openshift/configure-alertmanager-operator \
```

Comment:
Should this be a build-arg (`ARG`) in order to support referencing it in the Dockerfile? I'm not opposed to the hardcoding of it, but mainly curious.

Reply:
I tried. It didn't work.